services/meta/system.md

```diff
@@ -33,7 +31,35 @@ in   | cmd.node_reinterview        | str_map    | Value example `{"max_age":"1"}
 in   | cmd.channel.get             | null    | Requests the current Zigbee channel.
 out  | evt.channel.report          | int     | Reports the current Zigbee channel. The value is 0 if the network has not been established yet.
 
-#### Notes
+## Service props
+
+| Name                | Type   | Description                                                                                  |
+|---------------------|--------|----------------------------------------------------------------------------------------------|
+| `sup_configuration` | object | Optional. List of supported configuration params, see [`configuration_param`](#definitions). |
+
+## Definitions
+
+* `configuration_param` is an object of the following structure:
```

Comment:
@PaulinaDrozda Have you discussed this with @BudzikM ? How does it fit Zigbee usage?
Do we need size property? 
Format should be an enum with snake_case - spaces suggest a freeform text.
In case of checkbox, how do we know how many "options" are there to select and how to label them?

Reply:
Really important note. We're not making FIMP API a copy of ZWave or Zigbee spec. We abstract it for both and future specs too. Please keep this in mind.

Reply:
I would maybe need some device configuration, I can think of some generic settings like poll interval (wakeup interval) which all Zigbee device has and assumably ZWave ones as well. But we need to name it what it is and what it does so user could in a friendly way check how does it affect the device.